ext/dba/dba.c

```diff
@@ -724,7 +725,7 @@ static void php_dba_open(INTERNAL_FUNCTION_PARAMETERS, bool persistent)
 
 	info = pemalloc(sizeof(dba_info), persistent);
 	memset(info, 0, sizeof(dba_info));
-	info->path = pestrdup(ZSTR_VAL(path), persistent);
+	info->path = zend_string_dup(path, persistent);
```

Comment:
After looking at https://github.com/php/php-src/pull/9768, I think that we shouldn't use `zend_string_dup` when persistent might be `true`.

Reply:
This is confusing, but okay I see the issue :-/